packages/utils/src/flatMap.ts

```diff
@@ -1,5 +1,5 @@
 // Flattens at depth 1
-const flatten = (array: any[]) => {
+export const flatten = (array: any[]) => {
```

Comment:
Try to use `array.flat` if it exists and fallback otherwise.

Reply:
I would rather just use this until we bump the Chrome extension requirements, then we can remove this along with `fromEntries` etc..